ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -369,8 +403,19 @@ export default defineComponent({
       } else return '-'
     }
 
+    const getOwnerCount = (item: ManufacturedHomeSearchResultIF): string => {
+      const count = item.activeCount + item.exemptCount + item.historicalCount
+      if (count > 1) return `(${count})`
+    }
     const getItemClass = (item: ManufacturedHomeSearchResultIF): string => {
-      return item.selected && !props.isReviewMode ? 'selected' : ''
+      var rowClass = ''
+      if (props.isReviewMode && localState.uniqueResults < localState.results) {
```

Comment:
Why do we need to check if unique results are less than actual results?

Reply:
Unnecessary, forgot to remove. Good catch